src/js/letters/utils/constants.js

```diff
@@ -133,3 +133,9 @@ export const STATE_CODE_TO_NAME = {
   PI: 'Virgin Islands'
 };
 
+export const addressTypes = Object.freeze({
```

Comment:
Is it weird to add a constant variable that's not capitalized? I don't know too much about the conventions here, but that seems to be what we're following for the other constants.

Reply:
Yeah, that's fair. I can capitalize it. 👍